src/bindings/python/nixl-meta/nixl/__init__.py

```
@@ -17,20 +17,35 @@
+
+def _load_cuda_backend() -> str:
+    cuda_major = _get_torch_cuda_major()
+    if cuda_major is not None:
+        pip_name = f"nixl-cu{cuda_major}"
+        mod_name = f"nixl_cu{cuda_major}"
+        try:
+            return importlib.import_module(mod_name).__name__
+        except ModuleNotFoundError as e:
+            raise ImportError(
+                f"torch reports CUDA {cuda_major} but {pip_name} is not installed"
+            ) from e
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)